### DISCUSSION TITLE
battery 'Low_Background' argument

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [qtile/qtile#2668] [Ideas]
The battery low_background should be present, unlike low_foreground which is a difference in color of a few pixels , low_background is much better in letting you know that the battery is getting low.
I would love to see this simple feature implemented please.
Thanks!

**Comment by elParaguayo** [comment]
Are you able to have a go at submitting a PR for this?

**Reply by LamprosPitsillos** [reply]
How do i go about doing this?
